### PR TITLE
Earring/necklace/belt selection expansion

### PR DESCRIPTION
**Pull request by kyres1**
This expands the selection of earrings, necklaces and belts available in the loadout.

Earrings : bangles, large earrings, and crescent earrings. 

Colorable necklaces : rounded necklaces, small necklaces, low necklaces, and variations for those with pendants.

Colored necklaces : chain necklaces and small necklaces of gold and silver varieties, plus the existing necklace is still there.

Belts : Thin/normal/thick belts plus a buckle-belt.

Some examples below (not everything in the PR) : 
examples

### CODE REVIEW

**Comment by kyres1** [comment]
!review

**Review by alsoandanswer — COMMENTED** [review]
This PR looks pretty good. Just need the belts to be contained sprites and it's gucci.

**Review by alsoandanswer — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -591,7 +591,37 @@
 	gear_tweaks += new /datum/gear_tweak/path(temperature)
 
 /datum/gear/accessory/necklace
-	display_name = "necklace (colorable)"
-	description = "A piece of jewelry that goes around your neck."
+	display_name = "colored necklace selection"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm:594]:
```suggestion
	display_name = "necklace selection (colorable)"
```

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 2]
```diff
@@ -591,7 +591,37 @@
...
+	..()
+	var/list/colored = list()
+	colored["necklace"] = /obj/item/clothing/accessory/necklace
+	colored["golden"] = /obj/item/clothing/accessory/necklace/thin
+	colored["silver"] = /obj/item/clothing/accessory/necklace/thin/silver
+	colored["golden chain"] = /obj/item/clothing/accessory/necklace/chain
+	colored["silver chain"] = /obj/item/clothing/accessory/necklace/chain/silver
+	gear_tweaks += new /datum/gear_tweak/path(colored)
+
+/datum/gear/accessory/necklace_uncolored
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm:609]: isn't this supposed to be uncolourable?

File: code/modules/client/preference_setup/loadout/loadout_belt.dm [hunk 3]
```diff
@@ -1,17 +1,29 @@
 /datum/gear/belt
-	display_name = "belt"
+	display_name = "belt selection"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_belt.dm:2]:
```suggestion
	display_name = "belt selection (colorable)"
```

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 1 above)
Comment (kyres1) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:594]: colored here implies they are already colored in the icon and thus not colorable as an option. If there's a better way to word this (im sure there is) let me know

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 4]
```diff
@@ -591,7 +591,37 @@
...
+	display_name = "colorable necklace selection"
```
[the remark this answers is not in the document]
Comment (kyres1) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:610]: see above, but this is needed to distinguish the two imo

File: code/modules/client/preference_setup/loadout/loadout_belt.dm (hunk 3 above)
Comment (kyres1) [reply code/modules/client/preference_setup/loadout/loadout_belt.dm:2]: this is the only belt selection available so calling it colorable implies there's any alternative

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 1 above)
Comment (SleepyGemmy) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:594]: misunderstood the naming. it's fine as is.

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 4 above)
[the remark this answers is not in the document]
Comment (SleepyGemmy) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm:610]: misunderstood the naming. "necklace selection (colorable)" would work nicely.

File: code/modules/client/preference_setup/loadout/loadout_belt.dm (hunk 3 above)
Comment (SleepyGemmy) [reply code/modules/client/preference_setup/loadout/loadout_belt.dm:2]: it's mostly just to let people know its colourable and not just regular belts.

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**